hbase-client/src/main/java/org/apache/hadoop/hbase/shaded/protobuf/ProtobufUtil.java

```diff
@@ -3355,8 +3354,10 @@ public static String toLockJson(List<LockServiceProtos.LockedResource> lockedRes
     if (proto.hasOffline()) {
       rib.setOffline(proto.getOffline());
     }
-    if (proto.hasRegionEncodedName()) {
-      rib.setEncodedName(proto.getRegionEncodedName());
+    org.apache.hadoop.hbase.client.RegionInfo ri = rib.build();
+    if (proto.hasRegionEncodedName() && !proto.getRegionEncodedName().equals(ri.getEncodedName())) {
+      LOG.warn("The converted region info is {}, but the encoded name in proto is {}", ri,
```

Comment:
When this happen?

Reply:
The proto could be recevied from network so there is no guarantee that this will not happen?